manifests/blackbox-exporter-deployment.yaml

```diff
@@ -71,6 +77,13 @@ spec:
         ports:
         - containerPort: 9115
           name: https
+        resources:
```

Comment:
nit: I wonder why these weren't generated before?

Reply:
Because before adding kube-rbac-proxy (KRP) wasn't standardized and `deploymentMixin` didn't take into account global settings for KRP. Now it is added because I rewrote how KRP is added and it has a default setting for resources section as seen in https://github.com/prometheus-operator/kube-prometheus/blob/master/jsonnet/kube-prometheus/kube-rbac-proxy/container.libsonnet#L8-L11